### DISCUSSION TITLE
Dynamic access to Models

### DISCUSSION THREAD

**Discussion by koohz** [prisma/prisma#25439] [Q&A]
### Question

Sometimes in CRUD operations you need only to get the model from a string. I can't understand why its so difficult to get a model from string.

Somethin like 
```ts
db.getModel("Post").delete({ 
  where : {
    id : 1
  }
})
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @koohz 

Would the code below work for your usecase?

```js
const modelName = "post"; // This could be any model name as a string
const model = prisma[modelName]; // Access the model dynamically

const deletedPost = await model.delete({
  where: {
    id: 1,
  },
});
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
